py3status/modules/sysdata.py

```diff
@@ -3,18 +3,18 @@
 Display system RAM, SWAP and CPU utilization.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    format: output format string
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module
         *(default '[\?color=cpu CPU: {cpu_usage}%], '
         '[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]')*
     mem_unit: the unit of memory to use in report, case insensitive.
```

Comment:
>case insensitive

Fake news 📰. I added `case insensitive`. I think we'd be better off sticking with `KiB`, `MiB`, `GiB`, `auto`.